README.md

```diff
@@ -95,3 +95,12 @@ $ export WES_API_PROTO=http
 
 Then, when you call `wes-client` these defaults will be used in place of the
 flags, `--host`, `--auth`, and `proto` respectively.
+
+## Development
```

Comment:
Thanks for doing this. This repo needs more clear developer docs and you have to start somewhere!